### PR TITLE
[Fix] Correct format string for unw_word_t

### PR DESCRIPTION
**Pull request by ln5**
On x86_64 unw_word_t is uint64_t.

NOTE: Possibly incorrect for !x86_64.

Triggered by -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DENABLE_CLANG_PLUGIN=ON -DENABLE_LIBUNWIND=ON.

### CODE REVIEW

File: src/libserver/worker_util.c [hunk 1]
```diff
@@ -1639,11 +1639,11 @@ rspamd_print_crash(ucontext_t *uap)
 		ret = unw_get_proc_name(&cursor, name, sizeof(name), &off);
 
 		if (ret == 0) {
-			msg_err("%d: %p: %s()+0x%xl",
+			msg_err("%d: %ul: %s()+0x%xl",
 					level, ip, name, (uintptr_t) off);
```
Comment (vstakhov) [inline src/libserver/worker_util.c:1643]:
```suggestion
			msg_err("%d: 0x%xl: %s()+0x%xl",
					level, (unsigned long)ip, name, (unsigned long)off);
```

File: src/libserver/worker_util.c [hunk 2]
```diff
@@ -1639,11 +1639,11 @@ rspamd_print_crash(ucontext_t *uap)
...
 		}
 		else {
-			msg_err("%d: %p: <unknown>", level, ip);
+			msg_err("%d: %ul: <unknown>", level, ip);
```
Comment (vstakhov) [inline src/libserver/worker_util.c:1646]:
```suggestion
			msg_err("%d: %0x%xl: <unknown>", level, (unsigned long)ip);
```

**Review by vstakhov — APPROVED** [review]
This format strings are probably better

**Comment by vstakhov** [comment]
Hum, you have marked those comments as resolved but I don't see the changes here.

**Comment by ln5** [comment]
> Hum, you have marked those comments as resolved but I don't see the changes here.

Misunderstood the UI, committed now.

**Pull request merged**